02_using_sdram/README.rst

```diff
@@ -0,0 +1,98 @@
+Example 02: Reading and Writing SDRAM
+=====================================
+
+Most interesting SpiNNaker applications require some sort of configuration data
+to be loaded onto the machine or produce result data which must be read back
+from the machine. Typically this is done by allocating and writing/reading
+to/from the shared SDRAM on each SpiNNaker chip. In this example we'll write a
+simple SpiNNaker application which, using a single core, adds two numbers
+loaded in SDRAM and writes the answer back to SDRAM.
+
+Much of the code in this example is unchanged from the previous example so we
+will only discuss the cahnges.
+
+`Example Source code on GitHub <https://github.com/project-rig/rig_examples/tree/master/02_using_sdram>`_
+
+Allocating SDRAM
+----------------
+
+When both the host and SpiNNaker need to access the same block of SDRAM, such
+as when loading configuration data or reading back results, the SDRAM is
+typically allocated by the host.  In this example we'll allocate some SDRAM on
```

Comment:
"the SDRAM is typically allocated by the host" -> "it is common the host to allocate the SDRAM"?